### PR TITLE
Providing demo fiddle link for quick demo setup.

### LINKED ISSUE #3141: Add forkable fiddle in contribution template
### Description

 1. Guidelines suggest including demos for issues.

### Expected behavior

Provide a forkable fiddle link in the guidelines template with the bare minimum deps for using Mn. This will allow for quicker demo setup up.

### Environment

demo fiddle.

### PR DESCRIPTION
**Pull request by rafde**
### Providing demo fiddle link for quick demo setup.

closes #3141

### CODE REVIEW

**Comment by paulfalgout** [comment]
:+1:

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ Tell us what you think should happen.
 ### Actual behavior
 
 If possible, please create a small demo that demonstrates the issue.
+You can fork https://jsfiddle.net/marionettejs/adhv48ky/ for quick demo setup.
```
Comment (rafde) [inline ISSUE_TEMPLATE.md]: I didn't want to mention what versions of what the fork includes to make updating the fork easier and not have to continuously update this file for every upgrade.

I suggest a look at the external versions to make sure those are good... but like I said, easy to update.

Comment (denar90) [inline ISSUE_TEMPLATE.md]: What do you think about adding small note, that in fiddle used Mn v3, Radio v2 etc and if you are using another version, please don't forget to update resources in fiddle

**Comment by scott-w** [comment]
:+1:

**Pull request merged**